.github/workflows/pypi-release.yml

```diff
@@ -42,16 +42,18 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      - run: |
+      - name: Install pip requirements
+        run: |
           python -m pip install -U pip setuptools wheel build
-      - run: |
-          python -m build
-      - run: |
-          python -m pip install dist/*.whl
+          python -m pip install -r requirements.txt
+      - name: Setup and install manylinux1_x86_64 wheel
+        run: |
+          python setup.py bdist_wheel --plat-name=manylinux1_x86_64
```

Comment:
The correct way to produce manylinux wheels is by building in a manylinux Docker container, which has the appropriate older versions of system libraries. Here's an example from STT: https://github.com/coqui-ai/STT/blob/ebe189245517a45b92cdcfb1ba2eabbdd275840e/.github/workflows/build-and-test.yml#L217 (though that's manylinux2014, not manylinux1)

Reply:
By forcing the plat name like this, `pip` will allow the installation in compatible systems, but then users will get a crash at runtime due to missing symbols.